x-pack/plugins/security_solution/common/endpoint/types/trusted_apps.ts

```diff
@@ -35,7 +35,7 @@ interface MacosLinuxConditionEntry {
 type WindowsConditionEntry =
   | MacosLinuxConditionEntry
   | (Omit<MacosLinuxConditionEntry, 'field'> & {
-      field: 'signer';
+      field: 'porcess.code_signature';
```

Comment:
wait is this actually called porcess?

Reply:
Yeah, according to the info. I got :)

Reply:
Sorry - Just got what you were implying 😄 
It's a spelling mistake. Will correct it. Thanks for looking @parkiino